test/base/LendingPoolDepositTest.js

```diff
@@ -36,25 +37,42 @@ contract('LendingPoolDepositTest', function (accounts) {
     let settingsInstance;
     let marketsInstance;
 
+    const settingsMaxTotalValueLocked = 2000;
+
     beforeEach('Setup for each test', async () => {
         tTokenInstance = await Mock.new();
         daiInstance = await DAIMock.new();
         loansInstance = await Mock.new();
         interestConsensusInstance = await Mock.new();
         instance = await LendingPool.new();
         marketsInstance = await Mock.new();
-        settingsInstance = await Mock.new();
-        await settingsInstance.givenMethodReturnAddress(
-            settingsInterfaceEncoder.encodeMarketsState(),
-            marketsInstance.address
-        );
 
         cTokenInstance = await Mock.new();
         await cTokenInstance.givenMethodReturnAddress(
           cTokenEncoder.encodeUnderlying(),
           daiInstance.address
         )
 
+        settingsInstance = await createTestSettingsInstance(
+            Settings,
+            { 
+                from: accounts[0],
+                Mock,
+                initialize: true,
+                onInitialize: async(instance, { marketsState }) => {
+                    marketsInstance = marketsState;
+                }});
+          
+        await settingsInstance.createAssetSettings(daiInstance.address, cTokenInstance.address, 1000, { from: accounts[0] });
+
+        await settingsInstance.createPlatformSetting(
+            toBytes32(web3, settingsNames.MaximumTotalValueLocked),
+            2000,
```

Comment:
Should we consider the decimals in the calculation to get the max TVL?. There are no issues when there is only one market, but what will happen with two markets. i.e., DAI/ETH and USDC/ETH.

I.e., USDC has 6 decimals, and DAI has 18 decimals.

P.S.: There is an AssetSettings